GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -153,6 +147,81 @@ protected override void Execute(GVFSEnlistment enlistment)
             }
         }
 
+        private void CleanUpFoldersOutsideSparse(ITracer tracer, Enlistment enlistment, HashSet<string> sparseFolders)
+        {
+            string[] directoriesToDehydrate = this.GetDirectoriesOutsideSparse(enlistment.WorkingDirectoryBackingRoot, sparseFolders);
+            if (!this.ShowStatusWhileRunning(
+                () =>
+                {
+                    ReturnCode verbReturnCode = this.ExecuteGVFSVerb<DehydrateVerb>(
+                        tracer,
+                        verb =>
+                        {
+                            verb.Confirmed = true;
+                            verb.Folders = string.Join(";", directoriesToDehydrate);

```

Comment:
I notice that we have hardcoded the string literal used to separate directory entries in several places (thinking back to the original PR) - should we use a named constant for this?